scripts/apps/archive/views/preview.html

```diff
@@ -86,9 +89,12 @@
         <div class="rewrite" ng-if="selected.preview.rewrite_id">{{ :: 'This story has been rewritten by: ' | translate }}<a ng-click="previewRewriteStory()"
             class="open-item"><i class="icon-external"></i></a></div>
 
-        <p class="title preview-headline"
-           ng-if="selected.preview.headline && selected.preview.type !== 'composite'"
-           ng-class="{condensed: !lock}" sd-html-preview="selected.preview.headline"></p>
+        <p class="title z preview-headline"
```

Comment:
why `z`?

Reply:
:upside_down_face:  Forgot to get rid of that. There were 2 elements with the same class, I was testing which is getting rendered